src/tests/auth/userVerification.test.js

```diff
@@ -0,0 +1,37 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../server';
+import { user } from '../fixtures/user.fixture';
+import TokenService from '../../services/token.service';
+
+const token = TokenService.generateToken({
+	email: user.email
+});
+
+chai.should();
+chai.use(chaiHttp);
+
+describe('/GET email verification', () => {
```

Comment:
Remember to change this description method to `PATCH`

Reply:
Done